indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -153,13 +189,21 @@ export default function EditableList() {
                     />
                   )}
                 />
-                {columnHeaders.map(([key, label, width, extraProps]) => (
+                {code && (
```

Comment:
Maybe you could add this as a `columnHeader` conditionally somehow?
I couldn't find any pretty solution though...
Here's an example:
``` jsx
const columnHeaders = [
    ['id', Translate.string('ID'), 60],
    ...code ? [['code', Translate.string('Code'), 80]] : [],
    ['title', Translate.string('Title'), 600],
    ['status', Translate.string('Status'), 200],
    ['editor', Translate.string('Editor'), 400],
  ];
```

Reply:
It was looking a bit ugly but I can go ahead with this if everyone is fine with this. :+1: